index.d.ts

```diff
@@ -598,6 +601,20 @@ declare global {
     outlineStroke(): VectorNode | null
   }
 
+  interface MinimalFillsMixin {
+    fills: ReadonlyArray<Paint> | PluginAPI['mixed']
+    fillStyleId: string | PluginAPI['mixed']
+  }
+
+interface MinimalStrokesMixin {
+    strokes: ReadonlyArray<Paint>
+    strokeStyleId: string
+    strokeWeight: number
+    strokeJoin: StrokeJoin | PluginAPI['mixed']
+    strokeAlign: "CENTER" | "INSIDE" | "OUTSIDE"
+    dashPattern: ReadonlyArray<number>
+  }
```

Comment:
Instead of duplicating these properties, GeometryMixin could be a composition of the "Minimal" mixins

Reply:
Sure, I can also then do this will MinimalFillsMixin

Reply:
Done